components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/AbstractApplicationAuthenticator.java

```diff
@@ -189,7 +189,24 @@ protected boolean retryAuthenticationEnabled() {
 
     @Override
     public String getClaimDialectURI() {
-        return null;
+        String claimDialectUri = null;
+        AuthenticatorConfig authConfig = FileBasedConfigurationBuilder.getInstance().getAuthenticatorBean(getName());
+        if (authConfig != null) {
+            Map<String, String> parameters = authConfig.getParameterMap();
+            if (parameters != null && parameters.containsKey(FrameworkConstants.CLAIM_DIALECT_URI_PROPERTY)) {
```

Comment:
This method returns null if there is no ClaimDialectUri in authConfigs. So we don't need parameter.containsKey check here.